src/tagmanager.cpp

```diff
@@ -146,9 +146,7 @@ int TagManager::removeTag(Input input, Output output) {
     Monitor* monitor_target = find_monitor_with_tag(targetTag);
```

Comment:
That comment above this line is not really correct anymore is it?

Reply:
I've updated the comment.

Reply:
Thanks!